deepes/test/torch_agent/torch_agent_test.cc

```diff
@@ -0,0 +1,111 @@
+//   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Can we test the performance using the gtest framework ?

Reply:
done